web/src/main/java/org/cbioportal/web/util/ClinicalDataFilterApplier.java

```diff
@@ -46,13 +46,17 @@ public ClinicalDataFilterApplier(PatientService patientService,
             List<Patient> patients = patientService.getPatientsOfSamples(studyIds, sampleIds);
             List<String> patientIds = patients.stream().map(Patient::getStableId).collect(Collectors.toList());
             List<String> studyIdsOfPatients = patients.stream().map(Patient::getCancerStudyIdentifier).collect(Collectors.toList());
+            attributes.forEach(c-> {
+                c.setAttributeId(c.getAttributeId().toUpperCase());
```

Comment:
mysql queries are case insensitive. this won't be required

Reply:
@kalletlak , I do not think  ClinicalDataEqualityFilterApplier  is a right place to go. It overrides apply method of ClinicalDataFilterApplier and returns a total count instead of SampleIdentifier object.

Reply:
true, but the check whether the sample attribute value is in the filtered values values is done there

Reply:
right, need some changes in that class too. testing now. thanks!